pages/_type/_id/edit.vue

```diff
@@ -382,55 +379,103 @@
         />
       </label>
     </section>
-    <section class="card license">
-      <div class="title">
-        <h3>License<span class="required">*</span></h3>
-      </div>
-      <label>
-        <span>
-          It is very important to choose a proper license for your mod. You may
-          choose one from our list or provide a URL to a custom license.
-          <br />
-          Confused? See our
-          <a
-            href="https://blog.modrinth.com/licensing-guide/"
-            target="_blank"
-            rel="noopener noreferrer"
-            class="text-link"
-          >
-            licensing guide</a
-          >
-          for more information.
-        </span>
+    <section class="universal-card license">
+      <h3>License<span class="required">*</span></h3>
+      <div class="adjacent-input">
+        <label for="license-multiselect">
+          <span class="label__description">
+            It is very important to choose a proper license for your mod. You
+            may choose one from our list or provide a custom license. You may
+            also provide a custom URL to your chosen license; otherwise, the
+            license text will be displayed.
+            <br />
+            <span v-if="license && license.friendly === 'Custom'">
+              Enter a valid
+              <a
+                href="https://spdx.org/licenses/"
+                target="_blank"
+                rel="noopener noreferrer"
+                class="text-link"
+                >SPDX license identifier</a
+              >
+              in the marked area. If your license does not have a SPDX
+              identifier (for example, if you created the license yourself or if
+              the license is Minecraft-specific), simply check the box and enter
+              the name of the license instead.
+              <br />
+            </span>
+            Confused? See our
+            <a
+              href="https://blog.modrinth.com/licensing-guide/"
+              target="_blank"
+              rel="noopener noreferrer"
+              class="text-link"
+            >
+              licensing guide</a
+            >
+            for more information.
+          </span>
```

Comment:
Breaks aren't rendering for me for some reason. It looks like one giant text block. I think you may need to put double breaks so there's space in b/w paragraphs. would look a lot better